docker-compose.test.yml

```diff
@@ -40,7 +40,7 @@ services:
       service: service_base
     volumes_from:
       - static
-    command: py.test inspirehep tests/unit
+    command: ./run_tests.sh
```

Comment:
In order to run two commands in docker-compose.yml you have to write something like this:
 `command: bash -c "py.test inspirehep tests/unit && make -C docs html"`